src/assert.sh

```diff
@@ -249,3 +249,81 @@ function assert_is_file_empty() {
 
   State::addAssertionsPassed
 }
+
+function assert_directory_exists() {
+  local expected="$1"
+  local label="${3:-$(Helper::normalizeTestFunctionName "${FUNCNAME[1]}")}"
```

Comment:
Shouldn't this be using $2
```suggestion
  local label="${2:-$(Helper::normalizeTestFunctionName "${FUNCNAME[1]}")}"
```
Same applies to other assertions below where there is only one expected argument as $1, and the 2nd is the label.